src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-thu calendar--month-length-29">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--thu calendar--month-length--29">
```
before modifier value should be `--`